src/quality_result_gui/quality_errors_filters.py

```diff
@@ -339,12 +335,15 @@ class ErrorTypeFilter(AbstractQualityErrorFilter):
     _accepted_values: Set[QualityErrorTreeItemType]
 
     def __init__(self) -> None:
-        super().__init__(ERROR_TYPE_FILTER_MENU_LABEL)
+        super().__init__(self.get_error_type_filter_menu_label())
         self.menu.set_select_all_section_enabled(True)
         self.menu.set_sorted(True)
 
         self._refresh_filters(ERROR_TYPE_LABEL)
 
+    def get_error_type_filter_menu_label(self) -> str:
```

Comment:
If these were functions, you wouldn't have to use code like `ErrorTypeFilter.get_error_type_filter_menu_label(ErrorTypeFilter())` later on.